paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetResult.kt

```diff
@@ -22,10 +25,7 @@ sealed class CustomerSheetResult {
      */
     @ExperimentalCustomerSheetApi
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-    class Canceled internal constructor() : CustomerSheetResult() {
```

Comment:
We did this because it's easier to change a public API this way in the future. I think we should keep it!